doc/user_guide/metadata_structure.rst

```diff
@@ -0,0 +1,106 @@
+.. _metadata_structure:
+
+HoloSpy metadata structure
+**************************
+
+HoloSpy extends the :external+hyperspy:ref:`HyperSpy metadata structure
+<metadata_structure>`
+with conventions for metadata specific to its signal types. Refer to the
+:external+hyperspy:doc:`HyperSpy metadata documentation <user_guide/metadata_structure>`
+for general metadata fields.
+
+The metadata of any **signal objects** is stored in the `metadata` attribute,
+which has a tree structure. By convention, the node labels are capitalized and
+the ones for leaves are not capitalized. When a leaf contains a quantity that
+is not dimensionless, the units can be given in an extra leaf with the same
+label followed by the ``_units`` suffix.
+
+Besides directly accessing the metadata tree structure, e.g.
+``s.metadata.Signal.signal_type``, the HyperSpy methods
+:external:py:meth:`set_item() <hyperspy.misc.utils.DictionaryTreeBrowser.set_item>`,
+:external:py:meth:`has_item() <hyperspy.misc.utils.DictionaryTreeBrowser.has_item>` and
+:external:py:meth:`get_item() <hyperspy.misc.utils.DictionaryTreeBrowser.get_item>`
+can be used to add to, search for and read from items in the metadata tree,
+respectively.
+
+The holography specific metadata structure is represented in the following
+tree diagram. The default units are given in parentheses. Details about the
+leaves can be found in the following sections of this chapter. Note that not
+all types of leaves will apply to every type of measurement.
+
+::
+
+    metadata
+    ├── General
+    │   └── # see HyperSpy
+    ├── Sample
+    │   └── # see HyperSpy
+    ├── Signal
+    │   ├── signal_type
+    │   ├── quantity
+    │   └── # otherwise see HyperSpy
+    └── Acquisition_instrument
+        └── TEM
+            └── Biprism
+                ├── azimuth_angle (º)
+                ├── position
+                └── voltage (V)
+
+
+General
+=======
+
+See `HyperSpy-Metadata-General
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#general>`_.
+
+Sample
+======
+
+See `HyperSpy-Metadata-Sample
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#sample>`_.
+
+Signal
+======
+
+signal_type
+    type: string
+
+    String that describes the type of signal. The LumiSpy specific signal classes are
+    summarized under hyperspy:ref:`signal_types`.
+
+quantity
+    type: string
+
+    The name of the quantity of the “intensity axis” with the units in round brackets if
+    required, for example 'Intensity (counts/s)'.
```

Comment:
```suggestion
```

At least the sample datasets do not have this metadata field, otherwise we would need to know what a sensible example for the value is to adapt this.